infrastructure/backup-container/README.MD

```diff
@@ -0,0 +1,82 @@
+# Deployments
+
+<summary>Postgres deployment</summary>
+
+The following outlines the deployment of a simple backup of a PostgreSQL database in the 9c33a9-dev project namespace, on OCP v4.x.
+
+1. As per OCP4 [docs](https://developer.gov.bc.ca/OCP4-Backup-and-Restore), 5 of the storage class `netapp-file-standard` is the default quota. If this is insufficient, you may [request](https://github.com/BCDevOps/devops-requests/issues/new/choose) more.
+
+2. `git clone https://github.com/bcgov/moh-prime.git && cd infrastructure\backup-container`.
+
+Create the image and tag the image.
+
+```bash
+oc -n 9c33a9-tools process -f ./openshift/templates/backup/backup-build.yaml \
+  -p NAME=backup-container OUTPUT_IMAGE_TAG=dev| oc -n 9c33a9-tools create -f -
```

Comment:
Getting this error?

![image](https://github.com/bcgov/moh-prime/assets/7586665/9acfbbd9-8da5-434e-b614-d7b3fffa46e8)

Reply:
Did not `oc login`